src/providers/CustomModalsProvider.tsx

```diff
@@ -11,6 +11,9 @@ import { openAddToCollectionModal } from '~/components/Collections/AddToCollecti
 import { openManageHomeBlocksModal } from '~/components/HomeBlocks/ManageHomeBlocksModal';
 import { openBuyBuzzModal } from '~/components/Modals/BuyBuzzModal';
 import { openSendTipModal } from '~/components/Modals/SendTipModal';
+import BountyEntryFilesModal, {
```

Comment:
Let's do a dynamic import of the Modal component, just like we do it for all other modals
```suggestion
import {
```
Then down below
```ts
const BountyEntryFilesModal = dynamic(() => import('~/components/Bounty/BountyEntryFilesModal'));
```